drivers/wdm/audio/legacy/wdmaud/wdmaud.h

```diff
@@ -198,6 +198,13 @@ WdmAudMidiCapabilities(
     IN PWDMAUD_CLIENT ClientInfo,
     IN PWDMAUD_DEVICE_EXTENSION DeviceExtension);
 
+NTSTATUS
+NTAPI
+WdmAudGetPosition(
+    IN  PDEVICE_OBJECT DeviceObject,
+    IN  PIRP Irp,
+    IN  PWDMAUD_DEVICE_INFO DeviceInfo);
```

Comment:
```suggestion
    IN PDEVICE_OBJECT DeviceObject,
    IN PIRP Irp,
    IN PWDMAUD_DEVICE_INFO DeviceInfo);
```
either like I proposed, or (alternatively) SAL2, if the includes do allow that.

Reply:
Added SAL2 annotations here too.